### PR TITLE
tiflash: clarify signature of function date_add push down

### PR DESCRIPTION
**Pull request by LittleFall**
### What is changed, added or deleted? (Required)

tiflash: clarify signature of function date_add push down

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v6.2 (TiDB 6.2 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [x] v5.4 (TiDB 5.4 versions)

### CODE REVIEW

**Review by mengxin9014 — APPROVED** [review]
LGTM

File: tiflash/tiflash-supported-pushdown-calculations.md [hunk 1]
```diff
@@ -41,7 +41,8 @@ TiFlash 支持部分算子的下推，支持的算子如下：
 ## 下推限制
 
 * 所有包含 Bit、Set 和 Geometry 类型的表达式均不能下推到 TiFlash
-* date_add、date_sub、adddate 和 subdate 中的 interval 类型只支持如下几种，如使用了其他类型的 interval，TiFlash 会在运行时报错。
```
Comment (zanmato1984) [inline tiflash/tiflash-supported-pushdown-calculations.md:44]: 是不是可以只在这一行把`date_add``date_sub``adddate``subdate`签名写全就可以？不需要放在“下推限制”里？

Comment (zanmato1984) [reply tiflash/tiflash-supported-pushdown-calculations.md:44]: 忘了贴链接，这里：https://github.com/pingcap/docs-cn/pull/10726/files#diff-b8a560e5e7d65ea00606a4f3f51160c3ee35aca0fde623aa97381e8886335c7aL35

Comment (LittleFall) [reply tiflash/tiflash-supported-pushdown-calculations.md:44]: 好，之前主要是考虑那么写会重复太多，想提取出同类项。

**Review by zanmato1984 — APPROVED** [review]
LGTM

File: tiflash/tiflash-supported-pushdown-calculations.md [hunk 2]
```diff
@@ -32,7 +32,7 @@ TiFlash 支持部分算子的下推，支持的算子如下：
...
 * 位运算：`bitand, bitor, bigneg, bitxor`
 * 字符串函数：`substr, char_length, replace, concat, concat_ws, left, right, ascii, length, trim, ltrim, rtrim, position, format, lower, ucase, upper, substring_index, lpad, rpad, strcmp, regexp`
-* 日期函数：`date_format, timestampdiff, from_unixtime, unix_timestamp(int), unix_timestamp(decimal), str_to_date(date), str_to_date(datetime), datediff, year, month, day, extract(datetime), date, hour, microsecond, minute, second, sysdate, date_add, date_sub, adddate, subdate, quarter, dayname, dayofmonth, dayofweek, dayofyear, last_day, monthname, to_seconds, to_days, from_days, weekofyear`
+* 日期函数：`date_format, timestampdiff, from_unixtime, unix_timestamp(int), unix_timestamp(decimal), str_to_date(date), str_to_date(datetime), datediff, year, month, day, extract(datetime), date, hour, microsecond, minute, second, sysdate, date_add/adddate(datatime, int), date_add/adddate(string, int), date_add/adddate(string, real), date_sub/subdate(datatime, int), date_sub/subdate(string, int), date_sub/subdate(string, real), quarter, dayname, dayofmonth, dayofweek, dayofyear, last_day, monthname, to_seconds, to_days, from_days, weekofyear`
```
Comment (shichun-0415) [inline tiflash/tiflash-supported-pushdown-calculations.md:35]: This sentence contains `datetime` and `datatime`. Is this intended?

Comment (LittleFall) [reply tiflash/tiflash-supported-pushdown-calculations.md]: fixed, thanks

**Review by shichun-0415 — APPROVED** [review]
LGTM

**Review by LittleFall — COMMENTED** [review]
lgtm

**Pull request merged**